crates/config/src/config.rs

```diff
@@ -22,7 +22,7 @@ pub const VERSION_MESSAGE: &str = concat!(env!("CARGO_PKG_VERSION"));
 
 /// Protocol version that is used in anvil-zksync by default. Should match what is currently
 /// deployed to mainnet.
-pub const DEFAULT_PROTOCOL_VERSION: ProtocolVersionId = ProtocolVersionId::Version27;
+pub const DEFAULT_PROTOCOL_VERSION: ProtocolVersionId = ProtocolVersionId::Version28;
```

Comment:
@itegulov I suppose when we run our e2e tests it will run against v28 branch rather than v29. I tested locally with v29 and they pass but something we should consider updating in the CI to ensure they run against all supported versions.